pkg/services/apiserver/standalone/factory.go

```diff
@@ -33,6 +35,9 @@ type APIServerFactory interface {
 	// Given the flags, what can we produce
 	GetEnabled(runtime []RuntimeConfig) ([]schema.GroupVersion, error)
 
+	// Mixin for config
+	ConfigMixin(config *genericapiserver.RecommendedConfig, options *standaloneoptions.Options) *genericapiserver.RecommendedConfig
```

Comment:
I need something like this in addition to the `AddFlags` that this factory interface provides. That way, server config can be modified ahead of startup by enterprise code as needed.